packages/camera/camera/example/lib/main.dart

```diff
@@ -708,9 +708,10 @@ class _CameraExampleHomeState extends State<CameraExampleHome>
           // iOS only
           showInSnackBar('Audio access is restricted.');
           break;
-        case 'cameraPermission':
-          // Android & web only
-          showInSnackBar('Unknown permission error.');
+        case 'CameraPermissionsRequestOngoing':
+          // Android only
+          showInSnackBar(
+              'Another request is ongoing and multiple requests cannot be handled at once.');
```

Comment:
I think you can remove this case since it's an error that we do not expect developers to handle, which is different from exceptions.